clang/test/Driver/sycl-spirv-opt.cpp

```diff
@@ -0,0 +1,18 @@
+///
+/// Tests for -Xspirv-translator
+///
+
+// RUN: %clangxx -fsycl -Xspirv-translator "foo" -### %s 2>&1 | \
+// RUN:  FileCheck %s -check-prefix CHECK-SINGLE-TARGET
+
+// RUN: %clangxx -fsycl -Xspirv-translator=spir64_gen "foo" -### %s 2>&1 | \
+// RUN:  FileCheck %s -check-prefix CHECK-SINGLE-TARGET-UNUSED --implicit-check-not 'llvm-spirv{{.*}} "foo"'
+
+// RUN: %clangxx -fsycl -fsycl-targets=spir64,spir64_gen -Xspirv-translator=spir64 "foo" -Xspirv-translator=spir64_gen "bar" -### %s 2>&1 | \
+// RUN:  FileCheck %s -check-prefix CHECK-MULTIPLE-TARGET --implicit-check-not 'llvm-spirv{{.*}} "foo" "bar"'
+// CHECK-SINGLE-TARGET: llvm-spirv{{.*}} "foo"
+
+// CHECK-SINGLE-TARGET-UNUSED: argument unused during compilation: '-Xspirv-translator=spir64_gen foo'
+
+// CHECK-MULTIPLE-TARGET: llvm-spirv{{.*}} "foo"
+// CHECK-MULTIPLE-TARGET: llvm-spirv{{.*}} "bar"
```

Comment:
is there a way to verify that this particular call is associated with the `spir64_gen` side of the translation?

Reply:
ok should be fixed